components/slider/index.css

```diff
@@ -10,63 +10,94 @@ OF ANY KIND, either express or implied. See the License for the specific languag
 governing permissions and limitations under the License.
 */
 
-@import "../vars/css/components/spectrum-slider.css";
+
+
+/* missing tokens for variant with Tick, also need t-shirt sizing */
+.spectrum--medium {
+  --spectrum-slider-tick-mark-height: 10px;
+  /* --spectrum-global-dimension-size-125 */
+  --spectrum-slider-tick-mark-width: 2px;
+  /* --spectrum-global-dimension-size-25: */
+  --spectrum-slider-tick-mark-border-radius: 2px;
+  /* --spectrum-global-dimension-size-25 2px for Spectrum,   --spectrum-global-dimension-size-10 1px for Express */
+  --spectrum-slider-ramp-track-height: 16px;
+  /* dimension-size-200 */
+
+  
+}
+
+.spectrum--large {
+  --spectrum-slider-tick-mark-height: 13px;
+  --spectrum-slider-tick-mark-width: 2px;
+  --spectrum-slider-tick-mark-border-radius: 2px;
+  --spectrum-slider-ramp-track-height: 20px;
+}
```

Comment:
I _think_ we're going to need to pull these out of here, and place them into the Tokens package. I'll reach out to you about this on Slack.

Reply:
thanks, it is worth knowing that the Design team has not yet decided on the tokens for the tick and ramp variant as of now. That's why they were coded outside the token package @pfulton